tests/providers/amazon/aws/operators/test_sagemaker_processing.py

```diff
@@ -0,0 +1,153 @@
+
+job_name = 'test-job-name'
+
+create_processing_params = {
+    "AppSpecification": {
+        "ContainerArguments": ["container_arg"],
+        "ContainerEntrypoint": ["container_entrypoint"],
+        "ImageUri": "{{ image_uri }}",
+    },
+    "Environment": {"{{ key }}": "{{ value }}"},
+    "ExperimentConfig": {
+        "ExperimentName": "ExperimentName",
+        "TrialComponentDisplayName": "TrialComponentDisplayName",
+        "TrialName": "TrialName",
+    },
+    "ProcessingInputs": [
+        {
+            "InputName": "AnalyticsInputName",
+            "S3Input": {
+                "LocalPath": "{{ Local Path }}",
+                "S3CompressionType": "None",
+                "S3DataDistributionType": "FullyReplicated",
+                "S3DataType": "S3Prefix",
+                "S3InputMode": "File",
+                "S3Uri": "{{ S3Uri }}",
+            },
+        }
+    ],
+    "ProcessingJobName": job_name,
+    "ProcessingOutputConfig": {
+        "KmsKeyId": "KmsKeyID",
+        "Outputs": [
+            {
+                "OutputName": "AnalyticsOutputName",
+                "S3Output": {
+                    "LocalPath": "{{ Local Path }}",
+                    "S3UploadMode": "EndOfJob",
+                    "S3Uri": "{{ S3Uri }}",
+                },
+            }
+        ],
+    },
+    "ProcessingResources": {
+        "ClusterConfig": {
+            "InstanceCount": 2,
+            "InstanceType": "ml.p2.xlarge",
+            "VolumeSizeInGB": 30,
+            "VolumeKmsKeyId": "{{ kms_key }}",
+        }
+    },
+    "RoleArn": "arn:aws:iam::0122345678910:role/SageMakerPowerUser",
+    "Tags": [{"{{ key }}": "{{ value }}"}],
+}
+
+
+class TestSageMakerProcessingOperator(unittest.TestCase):
+
+    def setUp(self):
+        self.sagemaker = SageMakerProcessingOperator(
+            task_id='test_sagemaker_operator',
+            aws_conn_id='sagemaker_test_id',
+            config=create_processing_params,
+            wait_for_completion=False,
+            check_interval=5
+        )
+
+    def test_parse_config_integers(self):
+        self.sagemaker.parse_config_integers()
+        self.assertEqual(self.sagemaker.config['ProcessingResources']['ClusterConfig']['InstanceCount'],
+                         int(self.sagemaker.config['ProcessingResources']['ClusterConfig']['InstanceCount']))
+        self.assertEqual(self.sagemaker.config['ProcessingResources']['ClusterConfig']['VolumeSizeInGB'],
+                         int(self.sagemaker.config['ProcessingResources']['ClusterConfig']['VolumeSizeInGB']))
+
+    @mock.patch.object(SageMakerHook, 'get_conn')
+    @mock.patch.object(SageMakerHook, 'create_processing_job')
+    def test_execute(self, mock_processing, mock_client):
+        mock_processing.return_value = {'ProcessingJobArn': 'testarn',
+                                        'ResponseMetadata': {'HTTPStatusCode': 200}}
```

Comment:
You can directy set the `return_value` during _patching_.
```suggestion
    @mock.patch.object(SageMakerHook, 'create_processing_job', 
        return_value={ 'ProcessingJobArn': 'testarn', 'ResponseMetadata': { 'HTTPStatusCode': 200 }})
    def test_execute(self, mock_processing, mock_client):
```

More of these cases below.

Reply:
Fixed all cases of this.